platform/broadcom/one-image.mk

```diff
@@ -52,7 +52,9 @@ $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
                                $(ALPHANETWORKS_SNH60B0_640F_PLATFORM_MODULE) \
                                $(BRCM_XLR_GTS_PLATFORM_MODULE) \
                                $(DELTA_AG9032V2A_PLATFORM_MODULE) \
-                               $(JUNIPER_QFX5210_PLATFORM_MODULE)
+                               $(JUNIPER_QFX5210_PLATFORM_MODULE) \
+                               $(DELTA_AG9032V2A_PLATFORM_MODULE) \
```

Comment:
why add this line?

Reply:
Oh, these unrelated platform lines will be removed.

Reply:
> why add this line?

Oh, these unrelated platform lines will be removed.